web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -302,4 +318,12 @@ export default sfc; // activate IDE's Vue plugin
 .menu .item:hover .rss-icon {
   display: inline-block;
 }
+
+.scrolling.menu.is-loading {
+  height: 200px;
+}
+
+.scrolling.menu.is-loading::after {
+  display: block !important; /* to override fomantic rule .ui.dropdown .menu .menu:after {display: none} */
+}
```

Comment:
Could the full selector `.ui.dropdown .menu .menu:after` be used to replace the `!important` ?

Reply:
Yes changed to that https://github.com/go-gitea/gitea/pull/25719/commits/0eb8956cbcf998fdd74997cd09fa52c5632bed7e